### PR TITLE
add dnd_end_time in requestBody of updateStatus

### PR DESCRIPTION
**Pull request by darkLord19**
Server PR: mattermost/mattermost-server#16067

### CODE REVIEW

File: v4/source/status.yaml [hunk 1]
```diff
@@ -59,6 +59,9 @@
                 status:
                   type: string
                   description: User status, can be `online`, `away`, `offline` and `dnd`
+                dnd_end_time:
+                  type: integer
+                  description: Time on which dnd would be unset
```
Comment (agnivade) [inline v4/source/status.yaml:64]: Time in epoch seconds at which a `dnd` status would be unset.

**Review by jespino — APPROVED** [review]
Nothing to add to the @agnivade suggestion :)

**Comment by esethna** [comment]
@darkLord19 friendly ping on this PR so we can unblock the rest of them

**Comment by darkLord19** [comment]
@esethna this is not a blocker. QA review needs to be done for server and webapp pr and if it goes fine then all 3 PRs can be merged

**Review by agnivade — APPROVED** [review]

**Pull request merged**